app/controllers/dashboard_controller.rb

```diff
@@ -43,6 +43,8 @@ def autoyast
              Velum::SUSEConnect::MissingCredentialsException
         @do_registration = false
       end
+      ssh_key_file = "/var/lib/misc/ssh-public-key/id_rsa.pub"
+      @ssh_public_key = File.file?(ssh_key_file) ? File.read(ssh_key_file) : ""
```

Comment:
`@ssh_public_key = File.file?(ssh_key_file) ? File.read(ssh_key_file) : nil` might be a better fit. We can also check this `nil`/emptyness on the autoyast profile... see below.